src/vma/dev/qp_mgr.cpp

```diff
@@ -536,7 +536,12 @@ void qp_mgr::mlx5_send(vma_ibv_send_wr *p_send_wqe)
 	 * implementations may use move-string-buffer assembler instructions,
 	 * which do not guarantee order of copying.
 	 */
+#ifdef DEFINED_KVM_VMAPOLL
+	volatile uintptr_t* wqe_pt = (uintptr_t*)(m_sq_hot_wqe);
+	COPY_64B_NT_C(dst, wqe_pt);
+#else
 	COPY_64B_NT(dst, m_sq_hot_wqe);
+#endif
```

Comment:
@serglelv please think if we could have issue due to https://github.com/Mellanox/libvma/pull/262#discussion_r93785193
and
please consider doing `DEFINED_KVM_VMAPOLL` in COPY_^$ macro definition